mailbox/plugin/deleted-messages-vault/src/test/java/org/apache/james/vault/DeletedMessageZipperTest.java

```diff
@@ -82,6 +86,15 @@ void constructorShouldNotFailWhenCalledMultipleTimes() {
             }).doesNotThrowAnyException();
     }
 
+    @Test
+    void zipShouldThrowWhenPutMessageToEntryGetException() throws Exception {
```

Comment:
given the complexity of the error handling, I doubt you have the right coverage with this single test.